test/runnable/iasm64.d

```diff
@@ -3728,6 +3729,7 @@ void test50()
         int     0x67    ;
         //into          ;
 L10:    iret            ;
+        iretq           ;
```

Comment:
Not sure why the CIs other tha Win64 are not complaining, but by inserting the instruction here the jumps to L10 below change. It passes for me when moving `L10` to the `iretq` instruction.

Reply:
Okay, that's tricky. Should we instead move the label to a separate `nop` instruction and add a comment or would that require a new pull request?

Reply:
What's tricky bout it. Just change it to
```
        iret            ;
L10:    iretq           ;
```

BTW: I suspect these tests are run only on Win64 because they are disabled for version D_PIC at the top.

Reply:
The fix is not tricky, neither is the mistake I made, and of course I should have realised that the jumps change. The tricky part about the test case is that one has to realise this for oneself, which is why I would like to add a comment that explains what is going on, because in my opinion that's not something one *should* have to figure out.